src/garrysmod/gamemodes/zombiesurvival/gamemode/sv_database.lua

```diff
@@ -38,32 +40,19 @@ function GM.Database:Connect()
 end
 
 function GM.Database:Connected()
-    if self._Connected then
-        local _,is_open = pcall(function() return self.Connection:is_open() end)
-
-        -- If a connection has already been established, we can check the status
-        -- via pqxx::connection::is_open. Otherwise, use pcall to see if the "pg
-        -- - no connection, connect to a database first." error has been thrown.
-        if isbool(is_open) then
-            return is_open
-        else
-            return false
-        end
-    else
-        return false
-    end
```

Comment:
I was running into issues where it seemed like the act of invoking `pqxx::connection::is_open` actually closed the connection. Flux [does not do anything](https://github.com/TeslaCloud/flux-ce/blob/d5cf68d60c8db0ea40540786e0728a3fb42a1998/crates/active_record/lib/adapters/pg.lua) to keep the Postgres connection established, and I haven't ran into any instances where it closes on its own, so I'm just going to remove this for now.